### ISSUE TITLE
Połączenie z wind points

### ISSUE THREAD

**Issue by yoshomore** [pawel33332/DSJ-4-Wind-Unlocker#2]
Hej, czy ten mod jest w jakiś sposób połączony z wind points? Tzn. Gdybym odpalił konkurs z loteryjnymi warunkami, mógłbym mieć raz 4 m/s pod narty, a raz 5 m/s w plecy?

**Comment by pawel33332** [comment]
Nie, ten skrypt był na szybko zrobiony w ramach ciekawostki. Pozdrawiam.
